### PR TITLE
LTD-2592-part-number 

### PR DESCRIPTION
**Pull request by depsiatwal**
image

slightly different design but will confirm with Collin when he returns

### CODE REVIEW

File: exporter/applications/views/goods/common/payloads.py [hunk 1]
```diff
@@ -20,6 +20,19 @@ def get_pv_grading_details_payload(form):
     }
 
 
+def get_part_number_payload(form):
+    if form.cleaned_data["is_part_number"]:
```
Comment (saruniitr) [inline exporter/applications/views/goods/common/payloads.py:24]: if we provide spaces as part number then we still take the `if` path so unless API complains we will try to save it correct?

File: exporter/goods/forms/common.py [hunk 2]
```diff
@@ -216,3 +216,55 @@ def get_layout_fields(self):
                 render_to_string("goods/forms/common/help_with_security_gradings.html"),
             ),
         )
+
+
+class ProductPartNumberForm(BaseForm):
+    class Layout:
+        TITLE = "Do you have the part number ?"
```
Comment (saruniitr) [inline exporter/goods/forms/common.py:223]: there is an extra space before `?`

File: exporter/goods/forms/common.py [hunk 3]
```diff
@@ -216,3 +216,55 @@ def get_layout_fields(self):
...
+
+    is_part_number = forms.TypedChoiceField(
+        choices=(
+            (True, "Yes"),
+            (False, "No"),
+        ),
+        coerce=coerce_str_to_bool,
+        label="",
+        widget=forms.RadioSelect,
```
Comment (saruniitr) [inline exporter/goods/forms/common.py:232]: In the prototype links provided in ticket LTD-2592 there are no radio buttons on the form so just wondering are they not updated or not the correct links?

Comment (depsiatwal) [reply exporter/goods/forms/common.py:232]: correct please see my comment above we will confirm with collin when he returns.

Comment (saruniitr) [reply exporter/goods/forms/common.py:232]: :+1:

**Review by kevincarrogan — APPROVED** [review]

**Pull request merged**